lib/common-yargs.js

```diff
@@ -75,6 +74,13 @@ const withTitle = yargs => yargs.option("title", {
 	type: "string",
 })
 
+const withAccountType = yargs => yargs.option("account_type", {
+	alias: "a",
+	describe: 'The GitHub account type. Either "user", "org" (Organisation) or "repo" (Repository).',
+	type: "string",
```

Comment:
This should be restricting to choices if we are going with it.

http://yargs.js.org/docs/#api-choiceskey-choices